bin/start

```diff
@@ -29,7 +29,7 @@ EOF
   exit
 }
 
-function runScriptWithSummon() {
+runScriptWithSummon() {
```

Comment:
I prefer we will work with one [convention](https://google.github.io/styleguide/shellguide.html#naming-conventions) 
Lower-case, with underscores to separate words. (most of our code is written in this way)

Reply:
since I didn't write these I dont want to touch them too too much. It will become a huge PR

Reply:
this is already a huge PR. do you have an issue to fix it in another?

Reply:
It is in it's own commit so it is contained